test-util.ts

```diff
@@ -1,27 +1,40 @@
 /// <reference lib="deno.unstable"/>
 
+import { dirname, join } from "std/path/win32.ts";
+import { assertEquals } from "std/testing/asserts.ts";
+import { assertSnapshot } from "std/testing/snapshot.ts";
 import { Codec } from "./common.ts";
-export * as fixtures from "./target/fixtures/scale_fixtures.js";
 
-export const visitFixtures = <D, N>(
-  getFixtures: () => Map<Uint8Array, D>,
-  visit: (
-    bytes: Uint8Array,
-    decoded: N,
-    i: number,
-  ) => void,
-  normalize: (raw: D) => N,
-) => {
-  let count = 0;
-  for (const [bytes, decoded] of getFixtures().entries()) {
-    visit(bytes, normalize(decoded), count);
-    count += 1;
-  }
+const file = (path: string) => () => fetch(join(dirname(import.meta.url), path)).then((x) => x.text());
+export const files = {
+  lipsum: file("lipsum.txt"),
```

Comment:
Nitpick, why not...

```ts
const [lipsum, words, cargoLock] = ["lipsum.txt", "words.txt", "Cargo.lock"].map((fileName) => {
  return Deno.readTextFile(fileName);
});
export const files = { lipsum, words, cargoLock };
```

Using the path fns in combination with `fetch` feels less smooth.